german-iso.lbx

```diff
@@ -0,0 +1,16 @@
+\ProvidesFile{german-iso.lbx}
+[2016/05/15 v0.3.1 biblatex localization]
+
+\InheritBibliographyExtras{german}
+\InheritBibliographyStrings{german}
+
+\DeclareBibliographyStrings{%
+  at           = {{in}% maybe 'bei'
```

Comment:
`in` is definitely the wrong translation, but I’m not sure which one would be correct (`Ort`, `Exemplar`, …)